src/main/kotlin/no/nav/syfo/client/pdl/PdlClient.kt

```diff
@@ -0,0 +1,161 @@
+package no.nav.syfo.client.pdl
+
+import io.ktor.client.call.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.http.*
+import no.nav.syfo.application.cache.RedisStore
+import no.nav.syfo.client.azuread.AzureAdClient
+import no.nav.syfo.client.azuread.AzureAdToken
+import no.nav.syfo.client.httpClientDefault
+import no.nav.syfo.client.pdl.domain.*
+import no.nav.syfo.domain.PersonIdent
+import no.nav.syfo.util.*
+import org.slf4j.LoggerFactory
+
+class PdlClient(
+    private val azureAdClient: AzureAdClient,
+    private val pdlClientId: String,
+    private val pdlBaseUrl: String,
+    private val redisStore: RedisStore,
+) {
+    private val httpClient = httpClientDefault()
+
+    suspend fun personIdentNavnMap(
+        callId: String,
+        personIdentList: List<PersonIdent>,
```

Comment:
Vi ser av typen at det er en liste, hva med å kalle det personIdenter?

Reply:
`personIdenter` blir vel bare en norsk oversettelse av `personIdentList` ?

Reply:
Ikke egentlig, forslaget var ment å være flertall i stedet for liste. Jeg leste den som norsk i hodet, men ser jo at det kan være engelsk også (som er bedre 😁). Engelsk forslag er i så fall personIdents.

Reply:
Ja sant, da kom jeg med en kommentar på det under her.